content/docs/secure/encryption/rekey.md

```diff
@@ -0,0 +1,177 @@
+volumes. The general procedure for rekeying is to create a rekey custom resource via a simple yaml configuration. This 
+will kick off a rekey process on the PV specified as the `volume` in the resource. 
+
+## Rekey with dellctl
+
+If dellctl CLI is installed, rekey'ing a volume is simple. First, identify a volume with a PV that is encrypted with the CSM encryption driver
+provisioner. Verify that there is at least one volume attachment for this PV.
+
+For example, let's rekey an encrypted PV with the name ``K8s-asdf1234``:
+
+``dellctl encryption rekey myrekey k8s-asdf1234``
+
+```
+INFO rekey request "myrekey2" submitted successfully for persistent volume "k8s-112a5d41bc".
+INFO Run 'dellctl encryption rekey-status myrekey2' for more details.
+```
+
+Then to check the status of the newly created rekey with the name ``myrekey``:
+
+``dellctl encryption rekey-status myrekey``
+
+```INFO Status of rekey request myrekey2 = completed```
+
+See below for possible Status values and explanations.
+
+## Rekey with manually created Rekey CRs
+
+Identify a volume with a PV that is encrypted with the CSM encryption driver 
+provisioner. Verify that there is at least one volume attachment for this PV. 
+
+Now create a Rekey Custom Resource to start the rekey of a volume. For example, there is a PV with a name
+`k8s-09a76734f` as follows:
+```
+Name:            k8s-029a76734f
+Labels:          <none>
+Annotations:     pv.kubernetes.io/provisioned-by: sec-isilon.dellemc.com
+...
+Events:                <none>
+```
+
+An associated example Rekey CR for this volume:
+``` 
+apiVersion: "encryption.storage.dell.com/v1alpha1"
+kind: "Rekey"
+metadata:
+  name: "example-rekey"
+spec:
+  persistentVolumeName: "k8s-029a76734f"
+  ```
+
+Apply this CR yaml file to start the rekey process:
+
+```kubectl create -f my-example-rekey.yaml```
+
+### Inspect Status of Rekey
+
+Obtain the status of the rekey:
+
+```kubectl describe rekey my-example-rekey```
+
+```
+Name:         example-rekey
+Namespace:    
+Labels:       <none>
+Annotations:  <none>
+API Version:  encryption.storage.dell.com/v1alpha1
+Kind:         Rekey
+Metadata:
+  Creation Timestamp:  2022-11-11T14:01:51Z
+  Generation:          1
+  Managed Fields:
+    API Version:  encryption.storage.dell.com/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:spec:
+        .:
+        f:persistentVolumeName:
+    Manager:      kubectl-create
+    Operation:    Update
+    Time:         2022-11-11T14:01:51Z
+    API Version:  encryption.storage.dell.com/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:status:
+        .:
+        f:phase:
+    Manager:         rekey-controller
+    Operation:       Update
+    Subresource:     status
+    Time:            2022-11-11T14:01:51Z
+  Resource Version:  21062146
+  UID:               388a0245-18be-4e35-8618-814f30960b43
+Spec:
+  Persistent Volume Name:  k8s-029a76734f
+Status:
+  Phase:  completed
+Events:   <none>
+```

```

Comment:
Replace this with `kubectl get` which is less verbose:
```
# kubectl get rekey rekey4 -o yaml
apiVersion: encryption.storage.dell.com/v1alpha1
kind: Rekey
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"encryption.storage.dell.com/v1alpha1","kind":"Rekey","metadata":{"annotations":{},"name":"rekey4"},"spec":{"persistentVolumeName":"k8s-575b291c63"}}
  creationTimestamp: "2022-11-15T18:25:32Z"
  generation: 1
  name: rekey4
  resourceVersion: "23534992"
  uid: 81a85828-9777-4ffa-a25e-51a4416ff871
spec:
  persistentVolumeName: k8s-575b291c63
status:
  phase: rejected
```

Or better yet, just use this command to get the status:
```
echo $(kubectl get rekey rekey4 -o jsonpath='{.status.phase}')
```

Reply:
fixed as recommended.